Server reports XRandR version provided by libxrandr instead of XRandR
version implemented. It confuses applications that execute XRandR
version dependent code (e. g. gtk+ >= 2.18.0).

Index: xorg-server-1.3.0.0/randr/rrdispatch.c
===================================================================
--- xorg-server-1.3.0.0.orig/randr/rrdispatch.c
+++ xorg-server-1.3.0.0/randr/rrdispatch.c
@@ -22,6 +22,9 @@
 
 #include "randrstr.h"
 
+#define SERVER_RANDR_MAJOR	1
+#define SERVER_RANDR_MINOR	2
+
 Bool
 RRClientKnowsRates (ClientPtr	pClient)
 {
@@ -49,8 +52,8 @@ ProcRRQueryVersion (ClientPtr client)
      * Report the current version; the current
      * spec says they're all compatible after 1.0
      */
-    rep.majorVersion = RANDR_MAJOR;
-    rep.minorVersion = RANDR_MINOR;
+    rep.majorVersion = SERVER_RANDR_MAJOR;
+    rep.minorVersion = SERVER_RANDR_MINOR;
     if (client->swapped) {
     	swaps(&rep.sequenceNumber, n);
     	swapl(&rep.length, n);
